subworkflows/local/short_variant_calling/main.nf

```diff
@@ -70,18 +71,37 @@ workflow SHORT_VARIANT_CALLING {
 
     // Multisample
     GLNEXUS( ch_glnexus_in, ch_bed )
-    TABIX_GLNEXUS(GLNEXUS.out.bcf)
+
+    // Add allele count tag to multisample bcf
+    BCFTOOLS_FILLTAGS ( GLNEXUS.out.bcf )
+    ch_versions = ch_versions.mix(BCFTOOLS_FILLTAGS.out.versions)
```

Comment:
Let's try to be consistent and move the `ch_versions` statements to the end of the workflow

Reply:
Ah, I'm moving them to have them under every process, easier to miss at the end of the workflow 😄

Reply:
Sounds good! Mainly for consistency so that we don't have a couple after each process and some at the end

Reply:
Sure, I'll move all in this subworkflow then.